administrative-sdk/choiceRecognition.js

```diff
@@ -110,160 +94,148 @@ class ChoiceRecognition {
    *
    * @param {its.ChoiceChallenge} challenge The choice challenge to perform.
    * @param {its.AudioRecorder} recorder The audio recorder to extract audio from.
-   * @param {Sdk~choiceRecognitionPreparedCallback} [preparedCb] The callback that signals server is prepared for receiving data.
+   * @param {Sdk~choiceRecognitionPreparedCallback} [preparedCb] The callback that signals server is prepared for
+   *   receiving data.
    * @param {Sdk~choiceRecognitionCreatedCallback} [cb] The callback that handles the response.
    * @param {Sdk~choiceRecognitionCreatedErrorCallback} [ecb] The callback that handles the error response.
    * @param {Boolean} [trim] Whether to trim the start and end of recorded audio (default: true).
    */
-  startStreamingChoiceRecognition(connection, challenge, recorder, preparedCb, cb, ecb, trim) {
-    var self = this;
-    var _cb = function(data) {
-      var recognition = new ChoiceRecognition(
-        challenge.id, data.studentId, data.id,
-        new Date(data.created), new Date(data.updated),
-        connection.addAccessToken(data.audioUrl), data.recognised);
-      if (cb) {
-        cb(recognition);
-      }
-    };
-
-    var _ecb = function(data) {
-      // There was an unexpected error.
-      if (ecb) {
-        var analysis = new PronunciationAnalysis(
-          challenge.id, data.studentId, data.id,
-          new Date(data.created), new Date(data.updated),
-          connection.addAccessToken(data.audioUrl));
-        ecb(analysis, data.message);
-      }
-    };
-
-    // Validate required domain model.
+  startStreamingChoiceRecognition(connection, challenge, recorder, trim) {
```

Comment:
This method is pretty hard to follow... but I don't think it's this change is to blame.